libcst/tests/pyre/simple_class.json

```diff
@@ -0,0 +1,242 @@
+{
+  "types": [
+    {
+      "location": {
+        "path": "libcst/tests/pyre/simple_class.py",
+        "start": {
+          "line": 10,
+          "column": 6
+        },
+        "stop": {
+          "line": 10,
+          "column": 10
+        }
+      },
+      "annotation": "typing.Type[libcst.tests.pyre.simple_class.Item]"
+    },
+    {
+      "location": {
+        "path": "libcst/tests/pyre/simple_class.py",
+        "start": {
+          "line": 11,
+          "column": 8
+        },
+        "stop": {
+          "line": 11,
+          "column": 16
+        }
+      },
+      "annotation": "typing.Callable(libcst.tests.pyre.simple_class.Item.__init__)[[Named(self, unknown), Named(n, unknown)], unknown]"
```

Comment:
The type of self, n, and return of `__init__` are marked as unknown.
@shannonzhu